### ISSUE TITLE
Obsoletion request: collagen-containing extracellular matrix (GO:0062023)

### ISSUE THREAD

**Issue by DaiqingChen** [geneontology/go-ontology#29475]
Please provide as much information as you can:

* **GO term ID and Label**
collagen-containing extracellular matrix (GO:0062023)

* **Reason for deprecation** Put an x in the appropriate box: 
 - [x] The reason for obsoletion is that the term is not clearly defined and usage has been inconsistent.
 - [ ]  The reason for obsoletion is that this term is pre-composed and should be represented as a GO-CAM model.
 - [ ]  The reason for obsoletion is that this term represent an assay and not a GO process.
 - [ ]  The reason for obsoletion is that this represents a phenotype.
 - [ ] The reason for obsoletion is that the data from the paper for which the term was requested can be accurately described using [appropriate GO term].
 - [ ] The reason for obsoletion is that this term represents a molecular function.
 - [ ]  The reason for obsoletion is that this term represents a gene product.
 - [ ] The reason for obsoletion is that this term refers to a class of proteins.
 - [ ]  The reason for obsoletion is that this term was added in error.
 - [x]  The reason for obsoletion is that more specific terms were created.
 - [x]  The reason for obsoletion is that this term was an unnecessary grouping term.
 - [x]  The reason for obsoletion is that the meaning of the term is ambiguous.
 - [ ]  The reason for obsoletion is that is not known to be catalyzed by any gene product.
 - [ ]  The reason for obsoletion is that it more specific than the specificity of any known gene product.
 - [ ]  The reason for obsoletion is that there is no evidence that this function/process/component exists.

* **"Replace by" term (ID and label)**
child terms are re-assigned to different parents

* **"Consider" term(s) (ID and label)**
The new parent for child terms are as follow:
  * Child term moving to specialized ECM (#29267, NTR: #29265 )
     - Interphotoreceptor matrix (GO:0033165)
     - Cuticular ECM (GO:0060102)
     - Hyaline layer (GO:0033166)
     - Synapse-associated extracellular matrix (GO:0099535)
     - Extracellular matrix of synaptic cleft (GO:0098965) and perisynaptic extracellular matrix (GO:0098966) stay under synapse-associated extracellular matrix (GO:0099535) as they are.
  
  * Child term moving to non-collagenous component of interstitial matrix (#29267)
     - Matrilin complex (GO:0120216)
     - Tenascin complex (GO:0090733)
     - Transforming growth factor beta complex (GO:0099126) 
     - EMILIN complex (GO:1990971) 
     - Multimerin complex (GO:1990972) 
       
  * Child term moving to extracellular matrix (GO:0031012)(#29267)
     - Basement membrane (GO: 0005604)
     - Interstitial matrix (GO:0005614)
     - haven't tackle this part, not sure about localization, put under ECM for now.
     - All other annotations, unless otherwise specified, can be moved under extracellular matrix (GO:0031012)

  * Other child terms
     - Anchoring collagen complex (GO:0030934) obsoleted (#29457 )
     - Collagen type XXVI trimer (GO:1990324) move to collagenous component of interstitial matrix (NTR: #29473)
     - Collagen type XXVIII trimer (GO:1990326) move to collagenous component of basement membrane (NTR: #29472)
     - Complex of collagen trimer (GO:0098644) keep under Supramolecular complex (GO:0099080)
     
* **Are there annotations to this term?**
226,563 annotations

* **Are there mappings and cross references to this term? (InterPro, Keywords; check QuickGO cross-references section)**

* **Is this term in a subset? (check the AmiGO page for that term)**
No

* **Any other information**
See slide 10: https://docs.google.com/presentation/d/1v4o53iJN2Ywo1qi3L8ANZg-p6uCSVaGi/edit?usp=sharing&ouid=109267399050361021326&rtpof=true&sd=true
---
screenshot:

<img width="1483" alt="Image" src="https://github.com/user-attachments/assets/d41c7a17-5c53-4ad5-86dc-5a1a98f0dab0" />

----
Checklist for ontology editor

***Check term usage and metadata in Protégé***
- [ ] check term usage in the ontology 
- [ ] check internal mappings: RHEA, EC, MetaCyc
- [ ] check subset usage
- [ ] check taxon constraints 

***Check annotations***
- [ ] create Annotation Review spreadsheet 
- * Annotation Review template: https://docs.google.com/spreadsheets/d/1izV3q1jnq_Ybg5MJ53O_b5BCTkGtjl_Fi-cOg3-7eBE/edit#gid=0
- * Instructions: https://wiki.geneontology.org/Annotation_Review_Request
- [ ] create annotation review ticket: https://github.com/geneontology/go-annotation/issues/new?assignees=&labels=annotation+review&template=annotation-review-for-obsoletion.md&title=Review+annotations+to+%5BINSERT+GO+TERM%5D
- [ ] find external mappings (via IEAS), include in obsoletion notice  
- [ ] tag curators in Annotation Review ticket. Responsible annotator for each group: https://github.com/geneontology/go-site/blob/master/metadata/group-contacts.csv

***Notification***
- [ ] create [obsoletion announcement](https://github.com/geneontology/go-announcements/issues/new?assignees=&labels=obsoletion&template=obsoletion-notice.md&title=Obsoletion+notice%3A+%5BGO+ID%3A+term+label%5D)
- [ ] announce to GO friends (go-friends@mailman.stanford.edu)
- [ ] paste the text in the ontology ticket

**Comment by pgaudet** [comment]
1000's of annotations.

**Comment by pgaudet** [comment]
Dear all, 

Extracellular Matrix experts Alexandra Naba and Daiqing Chen have proposed to obsolete GO:0062023 collagen-containing extracellular matrix & replace it with parent GO:0031012  extracellular matrix. The reason for obsoletion is that the term was not clearly defined and usage has been inconsistent. There are also more specific terms that have been created.

For most annotations the replacement will work. There are 70 annotations that Alexandra recommended we review or remove, see https://github.com/geneontology/go-annotation/issues/5586

Impacted groups: 
BHF-UCL | 43
ComplexPortal | 1
MGI | 3
RGD | 5

as well as 9 InterPro and 1 UniRule.
Suggestions for fixing annotations have been made in the spreadsheet. 

You comment on the ticket: https://github.com/geneontology/go-ontology/issues/29475

Thanks, Pascale

**Comment by pgaudet** [comment]
@Matrisome 
I just now see that collagen-containing extracellular matrix (GO:0062023) has these part-of children: 



GO:0120216    matrilin complex | part_of
GO:0099126    transforming growth factor beta complex | part_of
GO:1990324    collagen type XXVI trimer | part_of
GO:0090733    tenascin complex | part_of
GO:0098644    complex of collagen trimers | part_of
GO:1990971    EMILIN complex | part_of
GO:1990972    multimerin complex

Should I move them to the parent extracellular matrix? 

Thanks, Pascale

**Comment by Matrisome** [comment]
@pgaudet Yes, all these can be moved to the parent "extracellular matrix" CC ontology (GO:0031012)

**Comment by DaiqingChen** [comment]
Hi Alexandra @Matrisome ,
&nbsp;&nbsp;Sorry I only noticed this change of parental relationship now, I'm not so sure whether we should put complex of collagen trimers (GO:0098644) part_of the extracellular matrix (GO:0031012) term. It is indeed part of the extracellular matrix, however its child are all under the children terms of extracellular matrix, and I believe removing this relationship will be more straightforward.
&nbsp;&nbsp;The collagen type XXVI trimer (GO:1990324) part_of extracellular matrix (GO:0031012) relationship has already been removed in ticket #30146 .

&nbsp;&nbsp;As I am working on the other complexes now, maybe we can keep as it is for now and come back to edits these relationship changes all together?
